var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -44,7 +44,8 @@ class Plasma(MakefilePackage):
     conflicts("veclibfort")
 
     # only GCC 4.9+ and higher have sufficient support for OpenMP 4+ tasks+deps
```

Comment:
maybe slightly reword this comment as it does not apply to both `conflicts`

Reply:
done